neo/test/iotest/test_nixio.py

```diff
@@ -1526,7 +1527,7 @@ def test_annotation_types(self):
             "sometime": time(13, 37, 42),
             "somequantity": self.rquant(10, pq.ms),
             "somestring": self.rsentence(3),
-            "npfloat": np.float(10),
+            "npfloat": float(10),
```

Comment:
here I think we are testing specific type. I would go for np.float64

Reply:
```suggestion
            "npfloat": np.float64(10),
```